sqldb/migrations_test.go

```diff
@@ -452,3 +452,121 @@ func TestCustomMigration(t *testing.T) {
 		})
 	}
 }
+
+// TestSchemaMigrationIdempotency tests that the our schema migrations are
+// idempotent. This means that we can apply the migrations multiple times and
+// the schema version will always be the same.
+func TestSchemaMigrationIdempotency(t *testing.T) {
```

Comment:
technically a migration such as migration 4 isnt idempotent by itself without a migration tracker right? but i dont think there is anything we can do about that. it is at least idempotent in the sense that if it is run multiple times in a row without moving past the migration run logic, then the net effect is zero since more invoices could not have been added

Reply:
Yeah I think it'd be actually safe to jump to version 4 and skip migration 4 altogether as we should assume that users on 0.18 are already on v0.18.5-beta.